.changeset/hot-apples-heal.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Maybe the comment was closed, PR titles should be more precise and meaningful
`feat: add hyperliquid core compatible Interchain token`